### PR TITLE
Add some notes on how to contribute

### PR DESCRIPTION
**Pull request by chenel**
Update the README with some thoughts on how to contribute.  Also add a CHANGELOG.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # `dune-plot-style` - DUNE official plot styling tools
 
 This repository contains coding tools to help analyzers easily make plots adhering to the DUNE Plot Style documented at https://wiki.dunescience.org/wiki/DUNE_Plot_Styles.
+(Those guidelines are also duplicated in this repository as [`guidelines.md`](https://github.com/DUNE/dune-plot-style/blob/main/guidelines.md).)
```
Comment (admlw) [inline README.md:5]: I would say to just keep the guidelines in github and link from elsewhere to here. Duplicated sets of guidelines are bound to get out of step with each other and it'll be a headache for people to figure out which is newer.

**Review by admlw — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -36,7 +38,7 @@ $ source /cvmfs/dune.opensciencegrid.org/products/dune/setup_dune.sh
 $ ups list -aK+ dune-plot-style
 
 # set up a specific version
-setup dune-plot-style v00.02
+setup dune-plot-style v00_02
```
Comment (admlw) [inline README.md:41]: I also wonder if we want to have the latest version setup by default - you can tag versions as `current` in ups. Can't think of many use cases we'd _not_ want the latest.

File: README.md (hunk 1 above)
Comment (chenel) [reply README.md:5]: Yeah, ok, that's a good call.

File: README.md (hunk 2 above)
Comment (chenel) [reply README.md:41]: Ah!  Yes.  Can you make sure that happens in the UPS tagging script?

File: README.md (hunk 1 above)
Comment (olantwin) [reply README.md:5]: And it would also be possible to directly publish the guidelines from there, as a nicer static website.

**Review by olantwin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chenel) [reply README.md]: eliminated the text about duplication in db323cf.

**Review by olantwin — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (chenel) [reply README.md:40]: @admlw ?

Comment (admlw) [reply README.md:40]: Sorry, yes. I don't think this needs to go into the script itself, they just need to use a couple of options when transferring to cvmfs. I can double check I'm remembering that and follow up with Jake about it today.

Comment (chenel) [reply README.md:40]: Ok.  If that turns out to be the case, let me know, and I'll merge this PR.

Comment (admlw) [reply README.md:40]: confirmed!

**Review by admlw — APPROVED** [review]

**Pull request merged**